detectors/sPHENIX/Fun4All_G4_sPHENIX.C

```diff
@@ -347,7 +347,17 @@ int Fun4All_G4_sPHENIX(
   //Additional tracking tools 
   //Enable::TRACKING_DIAGNOSTICS = Enable::TRACKING_TRACK && true;
   //G4TRACKING::filter_conversion_electrons = true;
+  // G4TRACKING::use_alignment = true;
 
+  // enable pp mode and set extended readout time
+  // TRACKING::pp_mode = true;
+  // TRACKING::pp_extended_readout_time = 20000;
+
+  // set flags to simulate and correct TPC distortions, specify distortion and correction files
+  //G4TPC::ENABLE_STATIC_DISTORTIONS = true;
+  //G4TPC::static_distortion_filename = std::string("/sphenix/user/rcorliss/distortion_maps/2023.02/Summary_hist_mdc2_UseFieldMaps_AA_event_0_bX180961051_0.distortion_map.hist.root");  
```

Comment:
Just as a comment, we should get these out of Ross' personal area and at least into the calibrations repo. Better yet would be to put them in CDBTree form and get them into the CDB